eslint-plugin-expensify/tests/prefer-double-negation.test.js

```diff
@@ -0,0 +1,77 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-double-negation');
+const message = require('../CONST').MESSAGE.USE_DOUBLE_NEGATION_INSTEAD_OF_BOOLEAN;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('use-!!-instead-of-Boolean()', rule, {
```

Comment:
```suggestion
ruleTester.run('use-double-negation-instead-of-boolean()', rule, {
```

Maybe name it the same as the rule so it is easy to find by the name of the rule? same comment for the file name to use `use-double-negation-instead-of-boolean.test.js` instead of `prefer-double-negation.test..js`